templates/windows_client.json.j2

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Any reason we can't use the existing https://github.com/sensu/sensu-ansible/blob/master/templates/client.json.j2 here? I see you added `environment` and `socket` to this one which I _think_ would make sense for everyone. Although, I would like to see `environment` a variable for folks deploying to staging/dev for example.

Reply:
I believe the socket part was required for windows. Not sure if it would have any undesirable effects for nix platforms as i have not tested it. I'll remove the environment for now since that's not an existing feature.

Reply:
`socket` should be fine cross platform as it's a standard `sensu-client` config, https://sensuapp.org/docs/1.2/reference/clients.html#socket-attributes and this is actually the default. Personally, I love the approach done on https://github.com/sensu/sensu-ansible/pull/147/files#diff-e887a90a9c451a87624736ecb4bdb48f where we can make a variable like `sensu_client_socket` and then use `to_nice_json` in the template. A rough example would be to add this global variable:
```
sensu_client_socket:
  bind: 127.0.0.1
  port: 3030
```
and then in the main template do:
```
"socket": {{ sensu_client_socket| to_nice_json(indent=6) }}
```
which should make your use case work and allow other users the flexibility to configure it. 

Does that see sane to you/